### PR TITLE
Updated signing rules and favicons, based on master 2021-04-18

### PR DESCRIPTION
**Pull request by dodmi**
Just merged the default rules and favicons from master branch as of 2021-04-18

### CODE REVIEW

File: data/signersDefault.json [hunk 1]
```diff
@@ -1,7 +1,5 @@
 {
 	"$schema": "./signersDefault-schema.json",
-	"versionTable": 1,
-	"versionData": 28,
```
Comment (lieser) [inline data/signersDefault.json:4]: This information should still be needed in the old version for the update of the default rules to apply.

As the rules are updated, the `versionData` number needs to be increased to `29`.

I would also leave `data/signersDefault-schema.json` unchanged.

**Review by lieser — CHANGES_REQUESTED** [review]
Note that I updated the rules again in master just now 640c482b5cb2d771fd7f7ada7c17a37388ae4ad8, which is not already in this branch.

**Comment by lieser** [comment]
Sorry for the noise, there has been another small update to the ruels and icons: 0ecc5764fd934509401883f5fbcb3eea0856992b

File: data/signersDefault.json (hunk 1 above)
Comment (dodmi) [reply data/signersDefault.json]: Thank you for your explanations; I've simply took the files from the master branch, as they looked similar.
Now, I've merged the changes and incremented the rules version as requested.

**Review by lieser — APPROVED** [review]

**Pull request merged**